### DISCUSSION TITLE
HL7 OPE^ZAx

### DISCUSSION THREAD

**Discussion by traujots** [nextgenhealthcare/connect#5103] [Q&A] [answered]
Hello,

[MIRTH 3.11]

I'll urgently try to implement some OPE-ZAx (ZA2 especially) for an anesthesia-program. Does Mirth support those messages? I see the Z0x - segments greyed out ... 
![ope-za2](https://user-images.githubusercontent.com/78225930/158989974-4cdffc92-cfea-467c-8388-0cad38261977.JPG)
[ope-za2.txt](https://github.com/nextgenhealthcare/connect/files/8303847/ope-za2.txt)

Thank you!

/Joerg

**Comment by jonbartels** [comment, marked as answer]
Yes MC supports Z-segments. You should be able to reference them with `msg['Z01']['Z01.2']['Z01.2.2']` like any other segment. They do not show up in the message template viewer because either Z-segments are not highlighted or the line ending after `PV1` is not correct.

You could also ask at the [Mirth Connect Slack ](https://forums.mirthproject.io/forum/mirth-connect/support/15720-join-the-mirth-connect-slack-channel). No response is guaranteed but the animals hanging around in the chat room may answer faster.

**Comment by traujots** [comment]
Hey Jon,

You saved my day!

Thank you and greetings from Germany

/Joerg

Von: jonbartels ***@***.***>
Gesendet: Freitag, 18. März 2022 13:59
An: nextgenhealthcare/connect ***@***.***>
Cc: Trautmann, Jörg ***@***.***>; Author ***@***.***>
Betreff: Re: [nextgenhealthcare/connect] HL7 OPE^ZAx (Discussion #5103)

This email originated from outside of CGM. Please do not click links or open attachments unless you know the sender and know the content is safe.


Yes MC supports Z-segments. You should be able to reference them with msg['Z01']['Z01.2']['Z01.2.2'] like any other segment. They do not show up in the message template viewer because either Z-segments are not highlighted or the line ending after PV1 is not correct.

You could also ask at the Mirth Connect Slack <https://forums.mirthproject.io/forum/mirth-connect/support/15720-join-the-mirth-connect-slack-channel> . No response is guaranteed but the animals hanging around in the chat room may answer faster.

—
Reply to this email directly, view it on GitHub<https://github.com/nextgenhealthcare/connect/discussions/5103#discussioncomment-2387588>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASU2ECSSMHJVVUTQOXBOHE3VAR4ZNANCNFSM5RBRNBQQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
